jumpscale/packages/admin/actors/admin.py

```diff
@@ -73,6 +73,10 @@ def get_identity(self, identity_instance_name: str) -> str:
         else:
             return j.data.serializers.json.dumps({"data": f"{identity_instance_name} doesn't exist"})
 
+    @actor_method
+    def get_current_identity_name(self) -> str:
+        return j.data.serializers.json.dumps({"data": j.core.identity.me.instance_name})
```

Comment:
`j.core.identity.me` is always set and cannot be None?

Reply:
yes threebot server can't start without default identity, and it will ask for it.

Reply:
can't it start in development mode anymore?